docbook5-book/xml/generic-entities.ent

```diff
@@ -156,6 +159,7 @@
 <!ENTITY salt         "Salt">
 <!ENTITY smaster      "&salt; Master">
 <!ENTITY sminion      "&salt; Minion">
+<!ENTITY kube         "Kubernetes">
```

Comment:
I wonder whether at least Salt/Master/Minion/Kubernetes should be moved to the System Components section (sorry for not reviewing that earlier).

Reply:
@sknorr reshuffled as requested